network/config/local-testnet.go

```diff
@@ -0,0 +1,43 @@
+package networkconfig
+
+import (
+	"math/big"
+	"time"
+
+	v1 "github.com/attestantio/go-eth2-client/api/v1"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	spectypes "github.com/ssvlabs/ssv-spec/types"
+)
+
+var LocalTestnetSSV = SSV{
+	Name:                      "local-testnet",
+	GenesisDomainType:         spectypes.DomainType{0x0, 0x0, spectypes.JatoV2NetworkID.Byte(), 0x1},
+	AlanDomainType:            spectypes.DomainType{0x0, 0x0, spectypes.JatoV2NetworkID.Byte(), 0x2},
+	RegistrySyncOffset:        new(big.Int).SetInt64(1),
+	RegistryContractAddr:      ethcommon.HexToAddress("0xC3CD9A0aE89Fff83b71b58b6512D43F8a41f363D"),
```

Comment:
I think the goal was to be able to pass in this specifically but everything.

Reply:
Yes, this structure is only used as default values in the `generate-config` CLI. I think it might be useful to omit some fields